src/igvfd/schemas/reference_file.json

```diff
@@ -129,6 +129,22 @@
         "schema_version": {
             "default": "16"
         },
+        "catalog_adapter": {
+            "title": "IGVF Catalog Adapters",
+            "comment": "IGVF Catalog Adapters that ingests this file",
```

Comment:
I think we can change "comment" to be a "description"  instead.

Reply:
good catch!